docs/content/page/index.zh-cn.md

```diff
@@ -67,7 +67,3 @@ Gitea的首要目标是创建一个极易安装，运行非常快速，安装和
 ## 软件及服务支持
 
 - [Drone](https://github.com/drone/drone) (CI)
```

Comment:
This line?

Reply:
Actions is an internal service but I think this is for 3rd party service. Maybe Jenkins and other CI/CD here.

Reply:
We might instead want to have docs include the "awesome list" of external tools